lib/galaxy/job_metrics/formatting.py

```diff
@@ -12,9 +12,25 @@ def seconds_to_str(value):
     """Convert seconds to a simple simple string describing the amount of time."""
     mins, secs = divmod(value, 60)
     hours, mins = divmod(mins, 60)
-    if value < 60:
+
+    if value == 0:
```

Comment:
Great! As a suggestion, you could simplify this, if you like. It's just 3 cases: value < 60; if not then value < 3600; and the rest - i.e., if/elif/else. As for the `s` suffix - you could pre-calculate it: it's `''` or `'s'` for secs/mins/hours depending on whether they `== 1`.